doc/news/changes.h

```diff
@@ -216,6 +216,14 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> New: added function GridOut::write_mesh_per_processor_as_vtu. This allows 
+ the visualization of a parallel finite element mesh that can be seperated into each 
```

Comment:
Oh, I clicked too early :-(

seperated -> separated.
Also maybe start the sentence after the colon with a capitalized "Added."

Feel free to propose a follow-up patch for this.